### ISSUE TITLE
Some pointless rambling for Friday

### ISSUE THREAD

**Issue by waryhermit** [ole-vi/BeLL-Apps#98]
# Some pointless rambling
I have been trying to do a better job managing lately, and I wanted write down some of my feelings.

### Overview of my feelings

1. The project feels cluttered, disorganized and lacking clear direction.
2. Its hard for me to figure out what issues are affecting end users or our ability to help people the most. I think my impression is clouded by what affects me and those I have to help.
3. Interns feel segregated and like they are more of a side project than an important part of getting the software where it needs to be.
4. Intern engagement is worse than I would like, probably because I'm struggling to figure out the best way to manage and how much I am supposed to be managing.

### Point 1 - Disorganization
I think some things that make me feel like the project is disorganized are the copious number of Github organizations and accounts used for the project, as well as the numerous derelict and out of date repositories. It seems like every time I work I end up at the wrong repository wondering why I can't find a certain pull request, issue or branch. I understand the need to organize things and keep them separate, but I feel like we don't have the right balance right now. I think the following changes would get rid of some of the annoyances I currently run into.

 - Move all inactive / backup repos off open-learning-exchange and on to some other account were they will be out of sight. You can read about [transferring repos here](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/).
 - Move every repository that is actively used or is actively being developed to the open-learning-exchange organization.
 - Merge open-learning-exchange/BeLL-Apps and ole-vi/BeLL-Apps and use branches to achieve separation and organization.

If waffle.io seems to cluttered after merging we can make better use of tags, and different groups of people can be given team names, then you can type a team name in the waffle filter an only issues tagged with a certain team will appear. Or another column could be added to separate teams, I suppose. But I am not we have enough people to need separation - and it sort of contributes to the feeling that interns have to be kept away from the people doing "real work".

### Point 2 - Lack of clear direction
I don't know if we just lack the information or if it just isn't being relayed to me. Do we receive any feedback from any of our deployments on what problems they have or features they would like? If we are it needs to be better integrated into our issues / waffle / management stuff.

If we are not getting any feedback, what are our plans to change this? It seems like if we could broaden our user base we might be able to get more feedback.

Another way we can help make the direction we want to go more clear is by using [Github milestones](https://help.github.com/articles/creating-and-editing-milestones-for-issues-and-pull-requests/) which is a helpful way to point out what order we want to do things in. It could also could be a morale booster, having well defined goals to reach for and achievements we can check off and look back at.

### Point 3 - Intern Environment
I think this is a side effect of 1 and 2. The more interns are able to help the developers work on "real issues" the more connected to the project and team members they will feel. The more it is just a endless cycle of one batch of interns helping the next batch of interns finish the first steps the more pointless it feels. It doesn't help that most of the pages about intern projects are unfinished and out of date.

I think having the http://talk.ole.org/ on all the time is a good step to helping with this problem, but we need to find a way to get people on it more often. 

### Point 4 - Motivation

I find it difficult to know what to assign people to work on, expecailly since many of the things I think need to be worked on the worst are issues on open-learning-exchange/BeLL-Apps where I am not allowed to assign and does not show up on waffle. And when people are not assigned anything to work on they have to muster up the motivation to pick something out of a sea of issues themselves. If I were able to assign them work they might feel like they were at least helping me or working on something someone thought was important.

Another thing we could do is have a little competition to see who can make the most commits in a month, not counting the markdown wiki. Perhaps have a little amazon gift card for the winner? Maybe that would make people less included to help each other though, so maybe not.

### Closing thoughts
I think the raspberry pi project exemplifies some of the things I have been talking about. Despite having the hardware and having talked to Dogi about it a few times I still don't know anything except that he doesn't want the toys to be abandoned but played with. I don't know what the end goal is or how the end goal will help the people OLE is trying to serve. Once I know what the end goal is and why we want that end goal I can be better able to work towards realizing the steps we need to reach that goal. Right now raspberry pi just seems like a hacking project with a dubious justification and enough backstory to make Moby Dick exciting.

All jokes aside, I think I will focus on figuring out what the heck is going on with the rasberry pi project this coming week, so I can at least understand it enough to even talk about it.

**Comment by waryhermit** [comment]
Regarding point 1 we are making the changes I suggested, except perhaps we are thinking more about merging the two BeLL-App repositories.

For point 2 we plan to make making issues / feedback part of the first steps for interns, and there was some talk about making a page listing the feedback we have got through the bell-apps / meetings with remote deployments. Not sure how @dogi felt about making milestones.. Though they are just special tags to make it more clear what has to be done before we make a new version.

For point 3 the plan is I will host a second hacking session at a different time of the week.

For point 4 we plan to make a little competition - but we have not decided on how we will measure the winner.
